nisystemlink/clients/dataframe/models/__init__.py

```diff
@@ -24,3 +24,6 @@
 from ._table_rows import TableRows
 
 # flake8: noqa
+
+# Alias to provide backwards compatibility for misnamed class, fixed in 1.0.2
+TableMetdataModification = TableMetadataModification
```

Comment:
I believe the linter wants a newline at the end of the file (build is failing).

```suggestion
TableMetdataModification = TableMetadataModification

```